docs/docs/tutorial/creating-a-source-plugin/part-1/index.mdx

```diff
@@ -0,0 +1,170 @@
+
+By the end of this part of the tutorial, you will have:
+
+- Cloned the companion repository to your local machine
+- Explored the project structure
+- Installed all required dependencies and built all packages
+- Spun up the local development server
+
+## Clone the repository
+
+Clone the [companion repository](https://github.com/gatsbyjs/creating-source-plugin-tutorial) from GitHub to your desired location. The repository's `main` branch shows the finished state after following this tutorial. You can reference its code as an additional resource while going through this guide.
+
+<Collapsible
+  summary={<em>GitHub cloning instructions</em>}
+>
+
+HTTPS snippet:
+
+```shell
+git clone https://github.com/gatsbyjs/creating-source-plugin-tutorial.git
+```
+
+SSH snippet:
+
+```shell
+git clone git@github.com:gatsbyjs/creating-source-plugin-tutorial.git
+```
+
+Also be sure to check out [GitHub's documentation for how to clone a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
+
+</Collapsible>
+
+After successfully cloning the repository, navigate to its folder and switch to the `start` branch to have a clean boilerplate:
+
+```shell
+git checkout start
+```
+
+Open the repository in your favorite editor as it'll make the next step easier.
+
+## Project structure
+
+The `creating-source-plugin-tutorial` project includes three directories:
+
+- `api`: This is the example backend API
+- `plugin`: The source plugin
+- `site`: An example Gatsby site
+
+So the `site` uses the `plugin` which sources its data from `api`. As mentioned in [Part 0](/docs/tutorial/creating-a-source-plugin/part-0/#development-environment) this project uses yarn workspaces to make this setup work.
+
+By having the plugin and an example site that uses the plugin in the same repository, you can more easily test your plugin.
+
+<Announcement>
+
+You can ignore the `api` folder unless you're curious how a [GraphQL Yoga server](https://github.com/dotansimha/graphql-yoga) is set up. For the purpose of this tutorial the example backend API was included so that this tutorial doesn't rely on outside resources. This way you can run everything locally.
+
+</Announcement>
+
+The `plugin` itself has a structure we'd also recommend following:
+
+- A `package.json` file that sets `gatsby` and `gatsby-plugin` as its `"keywords"` so that Gatsby's plugin search can pick it up
+- A root `gatsby-node.js` that requires `./dist/gatsby-node` (the compiled TypeScript file). At the moment Gatsby looks for a root `gatsby-node.js` file as its entry file.
```

Comment:
I noticed some of the list items here end with a '.' and some don't. Is it if there are more than one sentence, then a period is required? Just wondering what approach the Gatsby docs use!